docs/api/instruments/teledyne/index.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.teledyne
+
+#########
+Teledyne
+#########
+
+This section contains specific documentation on the Teledyne instruments that are implemented. If you are interested in an instrument not included, please consider :doc:`adding the instrument </dev/adding_instruments>`.
+
```

Comment:
That is, how you can connect to another documentation file with `:doc:'path_to_file_without_rst'` (you have to change the apostroph according to the example below). I made a suggestion of text:
```suggestion

You may find older devices under the name :doc:`../lecroy/index`.

```

Please add a similar comment to the `lecroy/index.rst` file, linking to this file.